governance/angle-dao.md

```diff
@@ -35,8 +35,12 @@ More formal proposals are then dicussed in our governance forum at [gov.angle.mo
 
 ## 🗳 Voting
 
-After proposals have been discussed, [Snapshot](https://snapshot.org/#/anglegovernance.eth/) votes are being opened to vote on these proposals. Since the tokenomics upgrade of January 2022, veANGLE holders are the ones with voting power over the protocol and whose balance is looked at when considering voting weights.
+After proposals have been properly discussed, they can be voted on through a [Snapshot](https://snapshot.org/#/anglegovernance.eth/) vote. After the tokenomics upgrade of January 2022, veANGLE holders are the ones with voting power over the protocol. As such, they are the ones voting to approve and implement or dismiss a proposal concerning the Angle Protocol.
 
-Finalized votes are implemented by a multi-sig composed of core team and community members.
+Last, the DAO is responsible for deciding where the liquidity mining rewards get distributed. Voting on this happens through takes place on [Snapshot](https://snapshot.org/#/anglegovernance.eth/) as well.
```

Comment:
This is no longer up to date. Gauge votes take place on-chain. We should say that only gauge votes for routing ANGLE emissions take place on-chain

Reply:
let me know if how I fixed this is fine